views/play1.erb

```diff
@@ -0,0 +1,7 @@
+<h1><%= @player1.name %> and <%= @player2.name %> entered the game!</h1>
+<h1><%= @player1.name %> choose your weapon!</h1>
+<form class="" action="/player1_choice" method="post">
```

Comment:
is class meant to be empty?

Reply:
yeah I didn't mean to leave it in there :)